core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -224,6 +225,63 @@ public void adddate() {
     assertEquals(missingValue(), eval(dsl.adddate(nullRef, missingRef)));
   }
 
+  @Test
+  public void convert_tz() {
```

Comment:
Andrew asked to create a separate file for each function and place each test (or test group) into separate method